etc/DependencyInstaller.sh

```diff
@@ -20,7 +20,7 @@ _installCommonDev() {
     spdlogVersion=1.8.1
 
     # temp dir to download and compile
-    baseDir=/tmp/installers
+    baseDir=${PREFIX}/tmp/installers
```

Comment:
I don't think you need to prefix `/tmp` because it is accessible by all users by default.

If you want to be extra careful, you can look at using `mktemp` which is  a shell utility that ensures that temporary filenames are unique (in the case of multi-user systems)